packages/platform/types/teams/outputs/team-members.output.ts

```diff
@@ -0,0 +1,26 @@
+import { Expose, Type } from "class-transformer";
```

Comment:
I would rename this file to `team-members-attributes.output.ts` since its not about / for team memberships per se.